src/ohow.ml

```diff
@@ -115,10 +122,35 @@ Overrides --print." in
       Arg.(value & opt (some string) None & info ["o"; "output"]
              ~docv:"FILE" ~doc)
     in
+    let cwd = Sys.getcwd () in
+    let root_cmd =
+      let doc = "Mark the given directory being the 'root directory', the directory that contains the version X of the documentation. The project directory must contain several root directories. The current directory by default." in
```

Comment:
Not sure to understand this explanation (and the following ones). Rephrase? And check english.

What is X?
"Mark ... as ..."?
The project directory must contain several root directories??
"The current directory by default." → ?

Reply:
See #44. I'll do it at the end (in case I change the behaviour of these options).